src/devices/Mcp23xxx/samples/Mcp23xxx.Sample.cs

```diff
@@ -81,13 +81,13 @@ private static Mcp23xxx GetMcp23xxxDevice(Mcp23xxxDevice mcp23xxxDevice)
             switch (mcp23xxxDevice)
             {
                 case Mcp23xxxDevice.Mcp23S08:
-                    return new Mcp23S08(s_deviceAddress, spiDevice);
+                    return new Mcp23s08(spiDevice, s_deviceAddress);
```

Comment:
My recommendation would be to add SpiMinAddress and SpiMaxAddress consts to all devices, so that users have an easier way to remember what the address should be

Reply:
As it is hardware configured I don't think it is worth it- you should never be programmatically needing it. We call it out in the comments, but you have to physically look at the wiring to know what address to use.